R/import_instruments.R

```diff
@@ -147,8 +147,12 @@ import_instruments <- function(url, token, drop_blank = TRUE,
   for (data_set in seq_len(n_instr_int)) {
     # all columns in the current instrument
     curr_instr_idx <- (big_i[data_set] + 1):big_i[data_set + 1]
-
-    drop_dot_one <- redcap[, c(meta, curr_instr_idx)] %>%
+    
+    column_index <-
+      c(meta, curr_instr_idx) %>%
+      unique()
+      
+    drop_dot_one <- redcap[, column_index] %>%
```

Comment:
This rewritten block collapses any duplicated.  I used a style similar to the one you used above: https://github.com/RaymondBalise/tidyREDCap/blob/d608422b173e27874d02dec62952c355e0a8c189/R/import_instruments.R#L56

Reply:
Is there an example where this causes a problem?  I think this would only happen if an instrument had no variables.

Reply:
yeah, something (I think `dplyr::select()`) was throwing an error.  The first column (ie, `record_id`) was being duplicated.  In some conditions, it was contained in both `meta` and `curr_instr_idx`.

Reply:
It would be excellent to see the example where this failed.  I need to see this code in action in a browser.  I think the unique may make other problems go away.